File: lucette2/src/main/java/battle/csn/lucette/service/InitGameHelper.java
package battle.csn.lucette.service;

public final class InitGameHelper {
	private InitGameHelper() {

	}

	public static String getGameId(String team) {
		return new RestClientAPI().getGameId(team);
	}

	public static String getTrainingGameId(String team, Integer level) {
		return new RestClientAPI().getNewPracticeGame(team, level);
	}
}

File: lucette2/src/main/scala/battle/csn/lucette2/client/InitGameHelper.scala
package battle.csn.lucette2.service

import _root_.battle.csn.lucette2.network.RestClient

object InitGameHelper {
  def getGameId(team:String)= {
    new RestClient().getGameId(team)
  }

  def getTrainingGameId(team:String , level:Integer )={
    new RestClient().getNewPracticeGame( team, level)
  }
}